vocabularies/Common.xml

```diff
@@ -1454,6 +1454,105 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <!--
+      <Term Name="RecommendationsFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#GetRecommendations))" />
+      </Term>
+      -->
+      <Function Name="GetRecommendations" IsBound="true">
```

Comment:
```suggestion
      <Function Name="AIRecommendations" IsBound="true">
```

Reply:
Then let's consistently prefix everything here with AI, and don't use HTTP verbs in function names.